### DISCUSSION TITLE
My website is not loading anymore with latest mobile version

### DISCUSSION THREAD

**Discussion by fpsweeper** [phantom/docs#176] [Help & Support] [answered]
### Summary

My web3 website was loading fine in all devices, but when i upgraded to latest phantom mobile version it stopped loading. this happens only on latest phantom mobile. It works on all other devices and web browsers and other wallets too, and prior phantom versions too.

### Example

https://rjctdlabs.xyz

### Steps to Reproduce

1. Visit https://rjctdlabs.xyz with phantom mobile version 24.1.1
2. That's it, the website will never loads.
(If you try a prior version 24.0.0 it works fine)

### Phantom Version

24.1.1

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hello @fpsweeper,

It appears your application now loads in the mobile browser. Please let me know if you are still seeing issues.

**Reply by fpsweeper** [reply]
Hello @adamdelphantom , thanks for your reply and sorry for my late reply. 
So the website now loads for some users but not loading for some others on the same mobile version.

**Reply by fpsweeper** [reply]
Also loaded for some people for a while then stopped. I am talking about version 24.2.0

**Reply by fpsweeper** [reply]
I don't know if this helps or not, but our website was flagged in URL:BLACKLIST and we just get it removed from there.

**Reply by therealMrFunGuy** [reply]
yeah @adamdelphantom 

we have many users still not able to view things. I was able to for about 24 hours that i noticed, but we seeing 50% of all mobile phantom users unable to access the site still. It seems like some days it connects, and somedays it doesn't at all for many days on end...

We're working with around 50 different web3 communities that are using the rjctdlabs.xyz site, so it's been really difficult between all our users. 

I'll help keep an eye on this chat so we can take care of any troubleshooting on our side if needed

**Reply by therealMrFunGuy** [reply]
fyi most users all on 24.2.0

**Reply by adamdelphantom** [reply]
@therealMrFunGuy @fpsweeper We haven't received any other reports like this from other applications. I can currently view the site reliably on iOS in Phantom.

From a debugging standpoint my best recommendation would be to create a local app with an instance of the [React Native Webview](https://github.com/react-native-webview/react-native-webview/tree/eb2ce07e728352abe8b11d10a9de2a4fdc2f228b) to try to get some insight from the console.

We are working on better debugging tooling for our mobile browser but I don't have an eta to share.

**Reply by therealMrFunGuy** [reply]
I guess that's the problem, is android and ios, some work, some dont... We've got probably 60+ people that it doesn't work at all for

I'll forward this to my team though thank you!

**Reply by therealMrFunGuy** [reply]
> @therealMrFunGuy @fpsweeper We haven't received any other reports like this from other applications. I can currently view the site reliably on iOS in Phantom.
> 
> From a debugging standpoint my best recommendation would be to create a local app with an instance of the [React Native Webview](https://github.com/react-native-webview/react-native-webview/tree/eb2ce07e728352abe8b11d10a9de2a4fdc2f228b) to try to get some insight from the console.
> 
> We are working on better debugging tooling for our mobile browser but I don't have an eta to share.

Yeah, this is really affecting people able to access our platform, and our projects expansion....

Makes no sense why it's only your mobile browser that doesn't work with our platform...

We've got a lot of people having phantom issues, more and more each day.


What is different from 24.0.0 to the Feb 6th update? What changed with how your app reads front end websites?

**Reply by therealMrFunGuy** [reply]
IOS seems to work fine for most, but definitly phantom on android having issues... its very hit or miss and clearly issues from your guy's updates

**Comment by therealMrFunGuy** [comment]
This is not fixed, we need dev assistance please...

**Comment by therealMrFunGuy** [comment]
![image](https://github.com/phantom/docs/assets/88522042/75d0a2a5-c932-4820-9e08-9f24274ecca0)

We can instantly import wallet on solflare and EVERYTHING works no issues...

**Reply by therealMrFunGuy** [reply]
@adamdelphantom 

Yeah, this is really affecting people able to access our platform, and our projects expansion....

Makes no sense why it's only your mobile browser that doesn't work with our platform...

We've got a lot of people having phantom issues, more and more each day.

What is different from 24.0.0 to the Feb 6th update? What changed with how your app reads front end websites?



solflare works no issues.... i'll tell 100k+ people to ditch phantom....

**Comment by therealMrFunGuy** [comment]
> @therealMrFunGuy @fpsweeper We haven't received any other reports like this from other applications. I can currently view the site reliably on iOS in Phantom.
> 
> From a debugging standpoint my best recommendation would be to create a local app with an instance of the [React Native Webview](https://github.com/react-native-webview/react-native-webview/tree/eb2ce07e728352abe8b11d10a9de2a4fdc2f228b) to try to get some insight from the console.
> 
> We are working on better debugging tooling for our mobile browser but I don't have an eta to share.



> @therealMrFunGuy @fpsweeper We haven't received any other reports like this from other applications. I can currently view the site reliably on iOS in Phantom.
> 
> From a debugging standpoint my best recommendation would be to create a local app with an instance of the [React Native Webview](https://github.com/react-native-webview/react-native-webview/tree/eb2ce07e728352abe8b11d10a9de2a4fdc2f228b) to try to get some insight from the console.
> 
> We are working on better debugging tooling for our mobile browser but I don't have an eta to share.

Maybe have someone look at your code that reads the front end of websites?

**Reply by adamdelphantom** [reply]
@therealMrFunGuy @fpsweeper I am sorry to hear your users are still having issues. We are continually working to improve the phantom mobile browser. If there was an inherent issue with the mobile browser we would see many other reports from other applications.

My best recommendation for your developers to be able to debug the issue is the above. 

Additionally you may have your users experiencing this contact Phantom support via Settings from their Phantom wallet.

**Reply by therealMrFunGuy** [reply]
> @therealMrFunGuy @fpsweeper I am sorry to hear your users are still having issues. We are continually working to improve the phantom mobile browser. If there was an inherent issue with the mobile browser we would see many other reports from other applications.
> 
> My best recommendation for your developers to be able to debug the issue is the above. 
> 
> Additionally you may have your users experiencing this contact Phantom support via Settings from their Phantom wallet.

That's the problem though fam... everything works for us on phantom 24.0.0

New versions broke things for us and we've not changed anything, nor do we have issues with our site access across the rest of web3..

I'll have all the communities start opening tons of tickets... but it's silly to tell us to debug when clearly the issues are coming from your mobile application updated on Feb 6th...

If you can ask someone to test with their android, or use an emulater with your different versions, you can see what I mean

**Comment by therealMrFunGuy** [comment]
> @therealMrFunGuy @fpsweeper We haven't received any other reports like this from other applications. I can currently view the site reliably on iOS in Phantom.
> 
> From a debugging standpoint my best recommendation would be to create a local app with an instance of the [React Native Webview](https://github.com/react-native-webview/react-native-webview/tree/eb2ce07e728352abe8b11d10a9de2a4fdc2f228b) to try to get some insight from the console.
> 
> We are working on better debugging tooling for our mobile browser but I don't have an eta to share.

@adamdelphantom @rawfalafel @justnom 

Currently we can sometimes force stop phantom, clear cache and then things work till next time we open phantom...


This site has issues too we have found...
https://one.juicevendor.com/RejectedY00ts/raid

Your mobile app is having issues with the cache....

**Comment by sineus** [comment]
Hi everybody,

I'm having the same issue with artrade.app (This bug impact some users). I have already proposed : 
- Clear app cache from android settings
- Reinstall app
- Reboot device

Nothing working, please help from @adamdelphantom  ...

**Reply by adamdelphantom** [reply]
Hey @sineus,

Please direct users experiencing this issue to write into support chat via settings in Phantom. Since this is isolated to certain devices we need to capture some information from the users directly.
